### DISCUSSION TITLE
Duplicate issues

### DISCUSSION THREAD

**Discussion by juancito-dev** [solodit/feedback#11] [Q&A]
There seems to by some duplicate issue like:

- https://solodit.xyz/issues/12261
- https://solodit.xyz/issues/12310

In this case, it is the exact same, except for `Submitted by gogo` vs `Submitted by HonorLt, gogo`

I haven't found a pattern yet, but will post it here if I do. Bringing this to attention in case there might be some issue with the indexer

**Comment by wang1252** [comment]
@0xJuancito 
We checked the two reports and confirmed they are exact same.
We will check other reports, too and remove the duplicates.
Thank you for your report.
